testdata/p4tc_samples_outputs/is_host_port_control_blocks.c

```diff
@@ -0,0 +1,306 @@
+#include "is_host_port_parser.h"
+struct p4tc_filter_fields p4tc_filter_fields;
+
+struct internal_metadata {
+    __u16 pkt_ether_type;
+} __attribute__((aligned(4)));
+
+struct __attribute__((__packed__)) ingress_nh_table_key {
+    u32 keysz;
+    u32 maskid;
+    u32 field0; /* hdr.ipv4.srcAddr */
+} __attribute__((aligned(8)));
+#define INGRESS_NH_TABLE_ACT_INGRESS_SEND_NH 1
+#define INGRESS_NH_TABLE_ACT_INGRESS_DROP 2
+#define INGRESS_NH_TABLE_ACT_NOACTION 0
+struct __attribute__((__packed__)) ingress_nh_table_value {
+    unsigned int action;
+    u32 hit:1,
+    is_default_miss_act:1,
+    is_default_hit_act:1;
+    union {
+        struct {
+        } _NoAction;
+        struct __attribute__((__packed__)) {
+            u32 port_id;
+            u64 dmac;
+            u64 smac;
+        } ingress_send_nh;
+        struct {
+        } ingress_drop;
+    } u;
+};
+
+static __always_inline int process(struct __sk_buff *skb, struct my_ingress_headers_t *hdr, struct pna_global_metadata *compiler_meta__)
+{
+    struct hdr_md *hdrMd;
+
+    unsigned ebpf_packetOffsetInBits_save = 0;
+    ParserError_t ebpf_errorCode = NoError;
+    void* pkt = ((void*)(long)skb->data);
+    u8* hdr_start = pkt;
+    void* ebpf_packetEnd = ((void*)(long)skb->data_end);
+    u32 ebpf_zero = 0;
+    u32 ebpf_one = 1;
+    unsigned char ebpf_byte;
+    u32 pkt_len = skb->len;
+
+    struct my_ingress_metadata_t *meta;
+    hdrMd = BPF_MAP_LOOKUP_ELEM(hdr_md_cpumap, &ebpf_zero);
+    if (!hdrMd)
+        return TC_ACT_SHOT;
+    unsigned ebpf_packetOffsetInBits = hdrMd->ebpf_packetOffsetInBits;
+    hdr_start = pkt + BYTES(ebpf_packetOffsetInBits);
+    hdr = &(hdrMd->cpumap_hdr);
+    meta = &(hdrMd->cpumap_usermeta);
+{
+        u8 hit;
+        u8 tmp;
+        u8 tmp_0;
+        {
+if (/* hdr->ipv4.isValid() */
+            hdr->ipv4.ebpf_valid) {
+                bpf_p4tc_is_host_port(skb, skb->ifindex);
```

Comment:
Also small detail, the return of `bpf_p4tc_is_host_port` is being lost

```suggestion
                tmp_0 = bpf_p4tc_is_host_port(skb, skb->ifindex);
```

Reply:
addressed